src/vorta/views/utils.py

```diff
@@ -2,14 +2,17 @@
 from vorta.utils import get_asset, uses_dark_mode
 
 
-def get_colored_icon(icon_name):
+def get_colored_icon(icon_name, force_color=''):
     """
     Return SVG icon in the correct color.
     """
     with open(get_asset(f"icons/{icon_name}.svg"), 'rb') as svg_file:
         svg_str = svg_file.read()
-    if uses_dark_mode():
-        svg_str = svg_str.replace(b'#000000', b'#ffffff')
+    if not force_color:
```

Comment:
It's generally better to avoid `if not` when one has a choice since `if-else` is easier to understand. So I suggest to swap this to be `if force_color:... else: ...`

Reply:
Done!